pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/custom/ArrayTest.java

```diff
@@ -873,7 +873,7 @@ public Schema createSchema() {
   }
 
   @Override
-  public File createAvroFile()
+  public List<File> createAvroFiles()
```

Comment:
Are the test changes related?

Reply:
yes

Reply:
I don't fully follow. Why do we change these tests?

Reply:
The previous test only create 1 segment, however to test reduce/merge, we need to create multiple segments per table.